R/SurvivalQuantities.R

```diff
@@ -356,3 +356,60 @@ setMethod(
         cat("\n", string, "\n\n")
     }
 )
+
+
+
+#' `brierScore`
+#'
+#' @description
+#' Derives the Brier Scores (using Inverse Probability of Censoring Weighting)
+#' for the Survival estimates as deatailed in Blanche et. al. (2015).
```

Comment:
```suggestion
#' for the Survival estimates as detailed in Blanche et. al. (2015).
```

Reply:
as next step / new issue could consider using `Rdpack` (see https://geobosh.github.io/Rdpack/ and https://github.com/openpharma/mmrm for example) for references such as this one